bin.js

```diff
@@ -26,34 +27,22 @@ function build () {
     ]
   } catch (_) {}
 
+  // run node-gyp-build
   proc.spawn(args[0], args.slice(1), { stdio: 'inherit' }).on('exit', function (code) {
-    if (code || !process.argv[3]) process.exit(code)
-    exec(process.argv[3]).on('exit', function (code) {
-      process.exit(code)
-    })
+    if (code) process.exit(code)
   })
 }
 
 function preinstall () {
-  if (!process.argv[2]) return build()
-  exec(process.argv[2]).on('exit', function (code) {
-    if (code) process.exit(code)
```

Comment:
`process.argv[2]` and `process.argv[3]` are used here to allow the user to specify commands that should be run before and after the build, respectively. It looks like this PR would currently remove that functionality.